### PR TITLE
API4 Update

### PR DESCRIPTION
**Pull request by keenanyafiqy**

### CODE REVIEW

**Comment by keenanyafiqy** [comment]
my quota is empty. you can merge it now if you want

**Comment by keenanyafiqy** [comment]
you can correct some code if somethings are wrong

File: BasicScore/plugin.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: BasicScore
 main: Ifera\BasicScore\Main
-api: [3.0.0]
-version: 1.2.1
+api: [4.0.0]
+version: 1.2.2
```
Comment (NhanAZ) [inline BasicScore/plugin.yml:4]: Do not arbitrarily change the plugin version

**Comment by Ifera** [comment]
Thanks for the PR but it has been updated already.

**Pull request closed without merging**